public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -248,10 +564,24 @@ function ModalMissionComplete ($, d3, L) {
                 // Update the horizontal bar chart to show how much distance the user has audited
                 var unit = "miles";
                 var regionId = neighborhood.getProperty("regionId");
+
+                // doing this the basic long way
+                var secondMax = 0;
+                var maxDist = 0;
+                var completedMissions = svl.missionContainer.getCompletedMissions();
```

Comment:
Please clean up this code. I recommend:

1. Filter out irrelevant missions first (i.e., completedMissions = completedMissions.filter( condition )).
2. Map the array of the filtered missions to an array of distance (i.e., distances = completedMissions.map( ... (extract distance) )
3. Sort the array of the extracted distances.
4. Check the length of the array of distances.
5. Do what you are doing now